packages/code-du-travail-data/indexing/buildThemes.js

```diff
@@ -40,9 +38,13 @@ function buildThemes(themes, getBreadcrumbs) {
             ({ position: positionA }, { position: positionB }) =>
               positionA - positionB
           )
-          .map(({ content: { slug, source, title } }) => ({
+          .map(({ content: { cdtnId, document, slug, source, title } }) => ({
```

Comment:
that's the change that makes getEsReferences useless, it should have been in the API commit, my bad :/